recipe/meta.yaml

```
@@ -23,12 +23,15 @@ requirements:
     - setuptools
   run:
     - python
-    - numpy >=1.22
+    - numpy >=1.22    # [py<311]
+    - numpy >=1.23.2  # [py==311]
+    - numpy >=1.26.0  # [py>=312]
     - opt_einsum
-    - scipy >=1.7
+    - scipy >=1.9  # [py<312]
+    - scipy >=1.11.1  # [py>=312]
     # Not declared in the metadata but essential to this package.
     # Update minimum from https://github.com/google/jax/blob/jaxlib-v0.3.25/jax/version.py
-    - jaxlib >=0.4.14
+    - jaxlib >=0.4.19
     - importlib_metadata >=4.6  # [py<310]
     - ml_dtypes >=0.2.0
```

```suggestion
    - ml_dtypes >=0.2.0
  run_constrained:
    - protobuf >=3.13,<4
```
https://github.com/google/jax/blob/jax-v0.4.23/setup.py#L108 ?